src/algebraic_geometry/structure_sheaf.lean

```diff
@@ -192,6 +192,89 @@ instance comm_ring_structure_sheaf_in_Type_obj (U : (opens (Spec.Top R))ᵒᵖ)
   comm_ring ((structure_sheaf_in_Type R).presheaf.obj U) :=
 (sections_subring R U).to_comm_ring
 
+open prime_spectrum
+
+lemma structure_sheaf_stalk_to_fiber_surjective (x : Top.of (prime_spectrum R)) :
+  function.surjective (stalk_to_fiber (is_locally_fraction R) x) :=
+begin
+  apply stalk_to_fiber_surjective,
+  intro t,
+  obtain ⟨r, ⟨s, hs⟩, rfl⟩ := (localization.of _).mk'_surjective t,
+  exact ⟨⟨⟨basic_open s, basic_open_open⟩, hs⟩, λ y, (localization.of _).mk' r ⟨s, y.2⟩,
+    ⟨prelocal_predicate.sheafify_of ⟨r, s, λ y, ⟨y.2, localization_map.mk'_spec _ _ _⟩⟩, rfl⟩⟩,
+end
+
+/--
+This follows the argument in Hartshorne.
+-/
```

Comment:
I can't imagine this being useful on hover. Maybe it should just be an ordinary comment?
```suggestion
/-
This follows the argument in Hartshorne.
-/
```
Or perhaps this should be expanded instead?